backend/apps/api/viewsets/community/__init__.py

```diff
@@ -84,6 +84,6 @@ def perform_create(self, serializer):
 
         creator = patched_request.user_profile
         # add creator as member and ranger of the community
-        community = serializer.save()
+        community: Community = serializer.save()
         community.members.add(creator)
         community.rangers.add(creator)
```

Comment:
```suggestion
        community.moderators.add(creator)
```
this is all we need actually.

Reply:
@moonlitgrace requested changes have been added.